tests/date/FixedDateRangeFormatting.test.ts

```diff
@@ -2,7 +2,7 @@ import { dateRange, FixedDateRange } from "@lib/date"
 
 describe("FixedDateRangeFormatting tests", () => {
   beforeEach(() => {
-    jest.useFakeTimers().setSystemTime(new Date("2023-02-26T12:00:00"))
+    jest.useFakeTimers("modern").setSystemTime(new Date("2023-02-26T12:00:00"))
```

Comment:
I was having trouble with it earlier since I was getting errors about the timers not being modern. You're right though, after I got rid of the errors I didn't check if I still needed to put "modern" in there.